README.md

```diff
@@ -1,2 +1,41 @@
-# 1fst-capstone-project
-This is my first capstone project into Microverse's program
+# My First Capstone Project
+
+> This is the best way that I can show you my work at: https://dangelo-jan.github.io/1fst-capstone-project/
+
+
+## Built With
+
+- HTML
+- CSS
+- Javascript
+
+## Getting Started
+
```

Comment:
[OPTIONAL]
- It would be nicer to add some steps on how the user can use your project.